warrior/Warrior

```diff
@@ -48,6 +49,77 @@ def update_jira_by_id(jiraproj, jiraid, exec_dir, status):
     else:
         print_info("jiraid not provided, will not update jira issue")
 
+def avg_time_calculate(abs_filepath, default_repo):
+    """ To print the Estimated Runtime for the Project/Suite/Testcase is
+    calculated using the previous junit file runtime values """
+
+    fname = abs_filepath.split('/')[-1].split('.')[0]
```

Comment:
/ or \ should not be used as they vary from operating system to another. use os.sep , this is a global comment for 
all your code

Reply:
@sugan19  use os.path.basename(abs_filepath).split('.')[0] to avoid using file separators.